mycroft/configuration/mycroft.conf

```diff
@@ -111,14 +111,28 @@
     "sample_rate": 16000,
     "channels": 1,
     "record_wake_words": false,
-    "wake_word": "hey mycroft",
-    "phonemes": "HH EY . M AY K R AO F T",
+    "record_utterances": false,
     "phoneme_duration": 120,
-    "threshold": 1e-90,
     "multiplier": 1.0,
-    "energy_ratio": 1.5
+    "energy_ratio": 1.5,
+     // wake word module
+    "wake_word_config": {
+        "module": "pocketsphinx",
+        "wake_word": "hey mycroft",
+        "phonemes": "HH EY . M AY K R AO F T",
+        "threshold": 1e-90,
+        "sample_rate": 16000
```

Comment:
Sample rate should probably be part of the listener config since there's no way to change that per hot word.

Reply:
changing